app/uk/gov/hmrc/emcstfe/services/GetMovementListService.scala

```diff
@@ -26,9 +28,10 @@ import javax.inject.{Inject, Singleton}
 import scala.concurrent.{ExecutionContext, Future}
 
 @Singleton
-class GetMovementListService @Inject()(connector: ChrisConnector) extends Logging {
+class GetMovementListService @Inject()(connector: ChrisConnector,
+                                       override val config: AppConfig) extends Logging with FeatureSwitching {
 
   def getMovementList(getMovementListRequest: GetMovementListRequest)
                      (implicit hc: HeaderCarrier, ec: ExecutionContext): Future[Either[ErrorResponse, GetMovementListResponse]] =
-    connector.postChrisSOAPRequestAndExtractToModel[GetMovementListResponse](getMovementListRequest)
+    connector.postChrisSOAPRequestAndExtractToModel[GetMovementListResponse](getMovementListRequest, isEnabled(ChrisStubEnabled))
```

Comment:
You won't need to have the fs here if you switch the url in app config, it'll be easier to remove when the time comes as well